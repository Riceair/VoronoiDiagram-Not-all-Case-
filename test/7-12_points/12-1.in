12
23 204
123 432
342 521
292 341
34 421
591 591
13 56
324 65
324 93
63 24
52 599
43 435