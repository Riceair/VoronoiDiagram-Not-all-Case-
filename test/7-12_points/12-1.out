P 13 56
P 23 204
P 34 421
P 43 435
P 52 599
P 63 24
P 123 432
P 292 341
P 324 65
P 324 93
P 342 521
P 591 591
E -240777.5 14092.8 -10827.51 862.8
E -112097 11850 -113.75 525.85
E -20122 -31460 92.32 124.97
E -10827.51 862.8 92.32 124.97
E -10827.51 862.8 92.94 309.23
E -113.75 525.85 81.7 400.2
E -113.75 525.85 86 514.89
E 81.7 400.2 86 514.89
E 81.7 400.2 92.94 309.23
E 86 514.89 197.61 562.34
E 92.32 124.97 171.29 142.52
E 92.94 309.23 151.96 283.34
E 151.96 283.34 193.29 202.18
E 151.96 283.34 242.59 451.67
E 171.29 142.52 188.04 79.17
E 171.29 142.52 193.29 202.18
E 188.04 79.17 947.75 79.16
E 188.04 79.17 26023.5 -164385.5
E 193.29 202.18 616.45 256.78
E 197.61 562.34 242.59 451.67
E 197.61 562.34 328.22 1047.92
E 242.59 451.67 517.29 375.38
E 328.22 1047.92 517.29 375.38
E 328.22 1047.92 5361.5 340165
E 517.29 375.38 616.45 256.78
E 616.45 256.78 947.75 79.16
E 616.45 256.78 947.75 79.16
E 947.75 79.16 331837.5 -167882
